.jvmopts

```diff
@@ -1,6 +1,9 @@
 -Dfile.encoding=UTF8
--Xms1G
--Xmx5G
--XX:ReservedCodeCacheSize=500M
--XX:+TieredCompilation
+-Xms2G
+-Xmx8G
```

Comment:
Nice. GitHub recently upgraded their runners so we have 16GB memory (I think the old limit was 7GB). We can't give it all to the JVM because Native (and JS) run their test suites as a separate process.

https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories